src/systems/systems/src/hydraulic/linear_actuator.rs

```diff
@@ -356,15 +370,26 @@ impl CoreHydraulicForce {
     }
 
     /// Computes a hydraulic flow request based on required actuator position and its current position
-    /// Flow is computed through the formula flow = position_error^3 * Gain
+    /// Flow is computed through the formula flow = position_error^2 * maxflow / OPEN_LOOP_POSITION_ERROR_FOR_MAX_FLOW^2
+    /// This formula means max flow will be applied above OPEN_LOOP_POSITION_ERROR_FOR_MAX_FLOW position error
+    /// Below OPEN_LOOP_POSITION_ERROR_FOR_MAX_FLOW position error, a squared law defines flow vs positioning error
+    ///
```

Comment:
**suggestion**: Refer to code elements such that `cargo doc` can detect broken documentation.

```suggestion
    /// Flow is computed through the formula flow = position_error^2 * maxflow / [Self::OPEN_LOOP_POSITION_ERROR_FOR_MAX_FLOW]^2.
    /// This formula means max flow will be applied above [Self::OPEN_LOOP_POSITION_ERROR_FOR_MAX_FLOW] position error.
    /// Below [Self::OPEN_LOOP_POSITION_ERROR_FOR_MAX_FLOW] position error, a squared law defines flow vs positioning error.
    ///
```

Reply:
The documentation does seem to be repeating what's in the code. It might be the documentation isn't adding any benefit over reading the code itself.